oat-sa/tao-core-libs-fe
# Upgrade lodash to 4.x

Ticket: https://oat-sa.atlassian.net/browse/ADF-1357

## What's Changed

- [lodash](https://www.npmjs.com/package/lodash) dependency been upgraded to 4.17.21 w/o locking the version

## How to test
- Run `npm run build` in order to check that build isn't broken
- Run `npm test` to check qunit-testrunner tests
